packages/css/src/hero/hero.scss

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../utils/breakpoint";
+
+@mixin reset {
+  box-sizing: border-box;
+  break-after: avoid;
+  break-inside: avoid;
```

Comment:
Are these two and `margin-inline` resetting default UA styles?

Reply:
I would say the reset mixin groups declarations that override unwanted values in the UA defaults. If we didn't have the mixin, we would be inclined to add a comment `/* UA override */` to each of these lines. Through the mixin, we move the ‘reset’ indicator higher up and group everything.

Our own design choices, for which the UA stylesheet specifies nothing or accepts the default value, should go in the declaration block for the component class name.